test/TM1.jl

```diff
@@ -52,6 +52,11 @@ end
         @test tv == TaylorModel1(5, 0.0, ii0)
         @test TaylorModel1(x1, 5, x0, ii0) == TaylorModel1(Taylor1(x1, 5), x0, x0, ii0)
         @test TaylorModel1(5, 0.7, ii1) == TaylorModel1(5, interval(0.7), ii1)
+        
+        @test TaylorModel1(tv, ii0) == 
+        TaylorModel1{Interval{Float64}, Float64}(Taylor1(Interval{Float64}, 5), ii0, x0, ii0)
```

Comment:
I think this test can fit in one single line:
```julia
@test TaylorModel1(tv, ii0) == TaylorModel1(Taylor1(Interval{Float64}, 5), ii0, x0, ii0)
```

Another possibility, which does not check all fieldnames, is
```julia
@test remainder(TaylorModel1(tv, ii0)) == ii0
```